src/Microsoft.AspNetCore.Razor.VSCode/src/RazorProject.ts

```diff
@@ -0,0 +1,27 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+import { IRazorProjectConfiguration } from './IRazorProjectConfiguration';
+
+export class RazorProject {
+    constructor(
+        public readonly uri: vscode.Uri,
+        public readonly path: string,
+        public readonly configuration: { [tfm: string]: IRazorProjectConfiguration }) {
+    }
+
+    public getCurrentConfiguration() {
+        let latestConfiguration: IRazorProjectConfiguration | undefined;
+        const allConfigurations = Object.values(this.configuration);
+        for (const projectData of allConfigurations) {
+            if (!latestConfiguration || latestConfiguration.lastUpdated < projectData.lastUpdated) {
+                latestConfiguration = projectData;
+            }
+        }
```

Comment:
I haven't spotted what value we get from tracking multiple configurations (keyed by TFM). It looks like we only ever use the most recently written one. Is there reason to think this will be needed later? If not then maybe there's a possible quick simplification.

Reply:
Ya good point, I was going to do more with the multiple configs but then went back on that idea. I'll change it to only track 1.